logic/src/commonMain/kotlin/com/wire/kalium/logic/AuthenticatedDataSourceSet.kt

```diff
@@ -0,0 +1,9 @@
+package com.wire.kalium.logic
+
+import com.wire.kalium.cryptography.ProteusClient
+import com.wire.kalium.network.AuthenticatedNetworkContainer
+
+class AuthenticatedDataSourceSet(
```

Comment:
This will later also hold a reference to an on disk persistence client?

Reply:
Something like that I guess.


I didn't think exactly how persistence can be packed, I guess one local persistence client/manager is necessary to perform DB migrations, provide independent scope per session, etc.


At some point we inject those into the repositories and have fun.